docs/docs/get-started/exploring-data/sharing-insights.md

```diff
@@ -13,28 +13,52 @@ In Lightdash there are three primary ways to share insights with your team:
 * Share a link to a saved chart
 * Download results as a .csv or a .png of the chart
 
-## 1. Share a URL to work in progress
+Before you do *any* sharing though, you need someone to share it with.
+
+## Inviting your teammates to your Lightdash project
+
+You can only share things with people you've invited to your Lightdash project.
+
+To invite your teammates, just click on `Settings` --> `Invite` --> `Create invite link`, then send your magic link to any of your teammates 🪄
+
+![invite user action](./assets/invite_user_action.gif)
```

Comment:
Should this still be a link to the gif?

Its causing an error in the build;

> 4:35:54 PM: Error: Image docs/get-started/exploring-data/assets/invite_user_action.gif used in docs/get-started/exploring-data/sharing-insights.md not found.

Reply:
ugh, okay yes, this was me trying to be sneaky and just merge something and then add pictures later